src/types/tmdb/tv.ts

```diff
@@ -176,3 +165,67 @@ export interface DiscoverTv {
     with_watch_providers?: string;
     watch_region?: string;
 }
+
+export interface Season {
+    air_date: string | null;
+    episode_count: number;
+    id: number;
+    name: string;
+    overview: string;
+    poster_path: string | null;
+    season_number: number;
+    vote_average: number;
+}
+
+export interface SeasonDetails extends Season {
+    _id: string;
```

Comment:
What is the difference here between `id` and `_id`? It seems potentially confusing that a SeasonDetail would have both, but maybe that is just how the API is structured?

Reply:
Okay cool. Thanks for clarifying. It seems like the id is what we would want to reference in the future if we needed to, and the _id is probably an internal UUID of some sort. No need to make any changes here, it just looked odd.